src/pages/summer/Main/Feed/Comments/Comments.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import './Comments.scss';
+
+class Comments extends React.Component {
+  render() {
+    const { comments, likes, handleClickDel, handleClickLike } = this.props;
+
+    return (
+      <ul className="detail-comments">
+        {comments.map((cmt, i) => (
```

Comment:
```suggestion
        {comments.map((comment, index) => (
```
여기도 마찬가지로 약어보다 명료하게 그대로 써주는게 좋을 것 같습니다!